velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -360,6 +360,15 @@ struct LastDayFunction {
   }
 };
 
+template <typename T>
+struct DateFromUnixDateFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(out_type<Date>& result, const int32_t& value) {
+    __builtin_add_overflow(0, value, &result);
```

Comment:
Please, document this behavior and include an example.

Reply:
Does this work?

Reply:
What's the purpose of __builtin_add_overflow here?

Reply:
I see what you mean, `result = value;` would work given the bounds.